stripe/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -539,73 +551,46 @@ class CardInputWidget @JvmOverloads constructor(
         cardNumberEditText.requestFocus()
     }
 
+    // reveal the full card number field
     private fun scrollLeft() {
         if (cardNumberIsViewed || !initFlag) {
             return
         }
 
-        val dateStartPosition = placementParameters.peekCardWidth +
-            placementParameters.cardDateSeparation
-        val cvcStartPosition = (dateStartPosition +
-            placementParameters.dateWidth + placementParameters.dateCvcSeparation)
+        val dateStartPosition = placementParameters.dateDestination
+        val cvcStartPosition = dateStartPosition + placementParameters.dateWidth + placementParameters.dateCvcSeparation
 
-        updateSpaceSizes(true)
+        updateSpaceSizes(isCardViewed = true)
 
-        val startPoint = (cardNumberEditText.layoutParams as FrameLayout.LayoutParams).leftMargin
-        val slideCardLeftAnimation = object : Animation() {
-            override fun applyTransformation(interpolatedTime: Float, t: Transformation) {
-                super.applyTransformation(interpolatedTime, t)
-                val params = cardNumberEditText.layoutParams as FrameLayout.LayoutParams
-                params.leftMargin = (startPoint * (1 - interpolatedTime)).toInt()
-                cardNumberEditText.layoutParams = params
-            }
-        }
+        val slideCardLeftAnimation = CardNumberSlideLeftAnimation(
+            view = cardNumberEditText
+        )
 
         val dateDestination = placementParameters.cardWidth + placementParameters.cardDateSeparation
```

Comment:
why is this no `placementParameters.dateDestination`?

Reply:
Fixed